src/core/ctxt.ml

```diff
@@ -3,65 +3,57 @@
 open Extra
 open Terms
 
-(** Representation of a typing context, associating a type (or [Term.term]) to
```

Comment:
I could probably have defined `type t = Terms.ctxt` here, but I guess what you did is fine. (This is just a comment, no need to change anything.)